modules/33acrossBidAdapter.js

```diff
@@ -179,6 +195,24 @@ function _createSync({ siteId = 'zzz000000000003zzz', gdprConsent = {}, uspConse
   return sync;
 }
 
+function _getBidFloors(getFloor, size) {
+  const bidFloors = getFloor({
+    currency: CURRENCY,
+    mediaType: MEDIA_TYPE,
+    size : [ size.w, size.h ]
+  });
+
+  if (!isNaN(bidFloors.floor) && (bidFloors.currency === CURRENCY)) {
+    return {
+      ext: {
+        ttx: {
+          bidfloors: [ bidFloors.floor ]
```

Comment:
wondering why do we send the `ext.ttx.bidfloors` but not the ortb standard bidfloor?

Reply:
nvm, i get it, this will do it for each single size